draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1715,6 +1715,12 @@ cookie-string from a given cookie store.
      *   The cookie's host-only-flag is false and the canonicalized
          host of the retrieval's URI domain-matches the cookie's domain.
 
+     NOTE: It's possible that the {{PSL}} has changed since a cookie was
+     created, potentially rendering the cookie now invalid if the
```

Comment:
I feel like "now invalid" is a bit imprecise. I don't think we define "invalid" anywhere. How about something along the lines of "such that the cookie would be ignored entirely in step 9 of the algorithm in section 5.5, if processed in the context of the updated public suffix list"?

Reply:
How's that?

Reply:
LGTM, thanks!